Applications/Tizen_CAPI/Tizen_CAPI_config.ini

```diff
@@ -0,0 +1,45 @@
+# Network Section : Network
+[Network]
+Type = NeuralNetwork	# Network Type : Regression, KNN, NeuralNetwork
+Layers = inputlayer \
+	 outputlayer	#Layers of Neuralnetwork
+Learning_rate = 0.0001 	# Learning Rate
```

Comment:
FYI, Note: Since Keras 2.2.4, the name will be replaced "Learning_rate" with "lr".

Reply:
Yes. we could support lr as well. I will add it later pr. Thanks.